tests/FunderMaps.Testing/Repositories/TestOrganizationUserRepository.cs

```diff
@@ -7,32 +7,26 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace FunderMaps.IntegrationTests.Repositories
+namespace FunderMaps.Testing.Repositories
 {
-    public class OrganizationUserRecord : BaseEntity<OrganizationUserRecord>
+    public class OrganizationUserRecord
```

Comment:
Separate file & folder?

Reply:
At best it be a internal entity/type. There may no need for a separate unit